Dockerfile

```diff
@@ -0,0 +1,62 @@
+#***************************************************************************************
+# Copyright (c) 2025 Beijing Institute of Open Source Chip (BOSC)
+#
+# XiangShan is licensed under Mulan PSL v2.
+# You can use this software according to the terms and conditions of the Mulan PSL v2.
+# You may obtain a copy of Mulan PSL v2 at:
+#          http://license.coscl.org.cn/MulanPSL2
+#
+# THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+# EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+# MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+#
+# See the Mulan PSL v2 for more details.
+#***************************************************************************************
+
+FROM verilator/verilator:latest
```

Comment:
Should we use the container built by xs-env? See https://github.com/OpenXiangShan/xs-env/pull/58

Reply:
we need a public available (host in dockerhub) xs-env image to inherit, then all these apt-get/mill `RUN` can be removed

Reply:
They are public available by `docker pull ghcr.io/openxiangshan/xs-env:latest`. See also https://github.com/OpenXiangShan/xs-env.

Reply:
we need a little update to (xs-env image)[https://github.com/OpenXiangShan/xs-env/pull/59]:
 - install verilator-wrapper.sh in verilator/ci/docker into /usr/local/bin
 - remove tmp file in RUN to reduce image size

Reply:
> @poemonsense @Tang-Haojin we need a little update to xs-env [OpenXiangShan/xs-env#59](https://github.com/OpenXiangShan/xs-env/pull/59):
> 
> * install verilator-wrapper.sh in verilator/ci/docker into /usr/local/bin
> * remove tmp file in RUN to reduce image size

merged